### PR TITLE
fix: enum variable usage error

### PR DESCRIPTION
**Pull request by wolfbolin**

### CODE REVIEW

**Review by hezhangjian — APPROVED** [review]

**Comment by hezhangjian** [comment]
I see unit test error, I think we need to support optional config first. @zhiheng123

File: opengemini_client/client_impl.py [hunk 1]
```diff
@@ -14,19 +14,20 @@
...
+from opengemini_client.models import AuthType
 
 
 def check_config(config: Config):
     if len(config.address) == 0:
         raise ValueError("must have at least one address")
 
     if config.auth_config is not None:
-        if config.auth_config.auth_type.PASSWORD == 0:
+        if config.auth_config.auth_type.PASSWORD == AuthType.PASSWORD:
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py]: do u mean that？
```
 if config.auth_config.auth_type == AuthType.PASSWORD:
```

Comment (wolfbolin) [reply opengemini_client/client_impl.py]: You are right

**Review by hezhangjian — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**